challenges/index.html

```diff
@@ -31,7 +31,7 @@
     >
       <div>Front-end development challenges</div>
       <div>
-        by
+        by Frontend Mentor
```

Comment:
This file shouldn't be changed in this branch.

Reply:
I have removed the text.